src/env.js

```diff
@@ -30,7 +30,7 @@ const VERSION = '3.5.2';
 
 // Check if various APIs are available (depends on environment)
 const IS_BROWSER_ENV = typeof window !== "undefined" && typeof window.document !== "undefined";
-const IS_WEBWORKER_ENV = typeof self !== "undefined"  && self.constructor?.name === 'DedicatedWorkerGlobalScope';
+const IS_WEBWORKER_ENV = typeof self !== "undefined" && (self.constructor?.name === 'DedicatedWorkerGlobalScope' || self.constructor?.name === 'ServiceWorkerGlobalScope' || self.constructor?.name === 'SharedWorkerGlobalScope');
```

Comment:
looks good to me, minor nit:
```suggestion
const IS_WEBWORKER_ENV = typeof self !== "undefined" && (['DedicatedWorkerGlobalScope', 'ServiceWorkerGlobalScope', 'SharedWorkerGlobalScope'].includes(self.constructor?.name));
```

@fs-eire to confirm, do these environments require anything specific for ort-web?

Reply:
could even do `self.constructor?.name.endsWith('WorkerGlobalScope')` but I'm not sure what other possibilities there are.

Reply:
This should work too, but I'm also not sure about other possibilities. That's why I went with exact matches, just to be safe.